src/components/InfoSidebar/InfoSidebar.css

```diff
@@ -0,0 +1,67 @@
+.info-sidebar-overlay {
+  background-color: #000;
+  opacity: 0.5;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  display: none;
+}
+
+.info-sidebar-overlay.show {
+  display: block;
+}
+
+.info-sidebar {
+  position: absolute;
+  right: 0px;
+  height: 235px;
+  display: flex;
+  top: 20%;
+  justify-content: flex-end;
+  transition: 300ms;
+}
+
+.info-sidebar__info-btn-container {
+  position: relative;
+  width: 40px;
+}
+
+.info-sidebar__info-btn-container__info-btn {
+  width: 40px;
+  background-color: white;
+  border-bottom-left-radius: 8px;
+  border-top-left-radius: 8px;
+  border: 2px solid black;
+  border-right: 2px solid white;
+  font-size: 1.5rem;
+  padding: 10px 9px 14px;
+  position: absolute;
+  top: 10%;
+  left: 2px;
+}
+
+.info-sidebar_content {
+  background-color: white;
+  border-bottom-left-radius: 8px;
+  border-top-left-radius: 8px;
+  border: 0px solid black;
+  border-right: 0;
+  width: 0;
```

Comment:
I would say that the width should always be `240px`.
Right now, when the sidebar opens, you'll notice a change in layout for the content. i.e. a jerk. That's because of the width being increased with transition.
We should keep the width at 240px, and just translate the content, or move its position.

i.e. the content is out of the screen, and then when we click the `i` button, the content comes in the screen.

Reply:
updated!